tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature

```diff
@@ -15,7 +15,21 @@ Feature: Sharing files and folders with internal users
     Given user "user2" has logged in using the webUI
     When the user shares folder "simple-folder" with user "User One" as "Viewer" using the webUI
     And the user shares file "testimage.jpg" with user "User One" as "Viewer" using the webUI
-    And the user re-logs in as "user1" using the webUI
+    Then user "user1" should have received a share with these details:
+      | field       | value              |
+      | uid_owner   | user2              |
+      | share_with  | user1              |
+      | file_target | /simple-folder (2) |
+      | item_type   | folder             |
+      | permissions | 1                  |
+    And user "user1" should have received a share with these details:
+      | field       | value              |
+      | uid_owner   | user2              |
+      | share_with  | user1              |
+      | file_target | /testimage (2).jpg |
+      | item_type   | file               |
+      | permissions | 1                  |
+    When the user re-logs in as "user1" using the webUI
     Then folder "simple-folder (2)" should be listed on the webUI
```

Comment:
We could have steps like:
```
Then as "user1" folder "simple-folder (2)" should be listed on the webUI
Then as "user1" folder "top/parent/child" should be listed on the webUI
```
The code for that would be remembering the currently-logged-in user. If the user is different then it can automatically do the "re-logs in as" actions. Then it browses down any parent folders (e.g. to `top/parent` if not already there) and then looks for `child`.

That would avoid the "random" `When the user re-logs in as` steps, and avoid having to also sometimes have steps that browse down into folders and sub-folders.

Is it worth doing like that? (In core also)

Reply:
@phil-davis 
implemented that in https://github.com/owncloud/phoenix/pull/1761/commits/b65c2134270121d8b66e6de754e62d00e23f8842

It clashes now a bit with `Then file "strängé नेपाली folder/lorem.txt" should be listed on the webUI`
`these resources should be listed on the webUI` navigates into the folder 
`should be listed on the webUI` does not but expects the file to be listed with the whole path (favorites)

Reply:
changed the behaviour now in a way that the folder has to be mentioned in the step itself to navigate into it
Still the user can be mentioned, so that a new login is carried out automatically and a list of items can be given to be checked